tensorflow-notebook-gpu/README.md

```diff
@@ -0,0 +1,15 @@
+[![docker pulls](https://img.shields.io/docker/pulls/jupyter/tensorflow-notebook.svg)](https://hub.docker.com/r/jupyter/tensorflow-notebook/)
+[![docker stars](https://img.shields.io/docker/stars/jupyter/tensorflow-notebook.svg)](https://hub.docker.com/r/jupyter/tensorflow-notebook/)
+[![image metadata](https://images.microbadger.com/badges/image/jupyter/tensorflow-notebook.svg)](https://microbadger.com/images/jupyter/tensorflow-notebook "jupyter/tensorflow-notebook image metadata")
+
+# Jupyter Notebook Deep Learning Stack
+
+GitHub Actions in the https://github.com/jupyter/docker-stacks project builds and pushes this image
+to Docker Hub.
+
+Please visit the project documentation site for help using and contributing to this image and
+others.
+
+- [Jupyter Docker Stacks on ReadTheDocs](http://jupyter-docker-stacks.readthedocs.io/en/latest/index.html)
+- [Selecting an Image :: Core Stacks :: jupyter/tensorflow-notebook](http://jupyter-docker-stacks.readthedocs.io/en/latest/using/selecting.html#jupyter-tensorflow-notebook)
+- [Image Specifics :: Tensorflow](http://jupyter-docker-stacks.readthedocs.io/en/latest/using/specifics.html#tensorflow)
```

Comment:
We should definitely add, which GPUs are supported.
It should be either list of GPUs or some kind of requirements.

Reply:
I think the main [requirement](https://docs.nvidia.com/datacenter/cloud-native/container-toolkit/install-guide.html) is CUDA and NVIDIA drivers must be installed on the host. As for GPUs, I think anything released in the last 10 years will work. If I look at the cuda-gpus [page](https://developer.nvidia.com/cuda-gpus) the lowest entry is the GeForce GT 430. Arguably, I think anybody looking to use tensorflow with GPUs will have a more modern GPU and (hopefully) have spent the time to make a good decision what GPU to get for their ML work.

Reply:
Having first link with the comment about host driver is fine for me.